test.js

```diff
@@ -35,22 +35,29 @@ test('efficiency', (t) => {
 
   t.test('whitespace around line', (t) => {
     const timeoutId = setTimeout(() => {
-      t.fail('did not pass in 10ms')
-    }, 10)
+      t.fail('did not pass in 30ms')
+    }, 30)
 
     t.deepEqual(
       trimLines(
-        'a' +
+        whitespace +
+          '\n' +
+          whitespace +
+          'a' +
           whitespace +
           '\n' +
           whitespace +
           'b' +
           whitespace +
+          'c' +
+          whitespace +
           '\n' +
           whitespace +
-          'c'
+          'd' +
+          whitespace +
+          '\n'
       ),
-      'a\nb\nc'
+      '\na\nb' + whitespace + 'c\nd\n'
```

Comment:
maybe make this two tests? One checking for the past thing, one checking for your new cases?

Reply:
To test different things: the previous problem you found, and the reason for this new change.